try{
console.log(a);
var a =13
welcome(12)
console.log(a);
const  welcome=(x)=>{
console.log("OkK Done");
}
}
catch(e){
    console.log("error");
}